RptToXml/Program.cs

```diff
@@ -58,28 +59,31 @@ static void Main(string[] args)
 				rptPaths.Add(rptPathArg);
 			}
 
-			foreach (string rptPath in rptPaths)
+			Parallel.ForEach(rptPaths, rptPath =>

```

Comment:
Sorry for taking so long to look at this!

Are the Crystal classes we use here thread-safe enough for this to be reliable? Have you tested against reports that have a most of the various object types populated?